pkg/controller/volume/persistentvolume/scheduler_binder.go

```diff
@@ -520,6 +545,89 @@ func (b *volumeBinder) isAllowedTopologySatisfied(class *storagev1.StorageClass,
 	return false
 }
 
+// isCapacitySatisfied check if capacity of the node domain in the
+// storage class can satisfy resource requirement of given claim,
+// and return its value of AnnProvisionedTopology if needed.
+func (b *volumeBinder) isCapacitySatisfied(claim *v1.PersistentVolumeClaim, class *storagev1.StorageClass, node *v1.Node, usedCache map[string]CapacityPerClass) (minSizeSatisfied bool, capacitySatisfied bool, provisionedTopologyValue string) {
+	if class.Status == nil || class.Status.Capacity == nil {
+		// No capacity limitation, return true
+		return true, true, ""
+	}
+
+	// Requested resource by the claim
+	requested := claim.Spec.Resources.Requests[v1.ResourceStorage]
+	minVolumeSize := class.Status.Capacity.MinVolumeSize
+
+	if !isMinVolumeSizeSatisfied(requested, minVolumeSize) {
```

Comment:
I misunderstood this part of `MinVolumeSize` in design, and thought we need to reject it if a user requested a volume under the minimum size.